pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -375,6 +384,22 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 	ctx.subVolumesSnapStatus = subVolumesSnapshotStatus
 	_ = subVolumesSnapshotStatus.Activate()
 
+	ctx.subAssignableAdapters, err = ps.NewSubscription(pubsub.SubscriptionOptions{
```

Comment:
This probably is not needed if lookupDomainConfig() is used.

Reply:
1. As I explained before, the moment when we evaluate the memory requirements is earlier than the moment when DomainConfig is created.
2. We still need to have a list of assignable adapters that are stored per system, not per domain. As far as I understand we need this system-wide list to check against it the list of the adapters assigned in the domainConfig. @rucoder should know this logic. I'll recheck how it's used. So we have to get this list in zedmanager.